refinery/core/utils.py

```diff
@@ -983,8 +983,7 @@ def move_obj_to_front(obj_arr, match_key, match_value):
 
 
 def api_error_response(error_message, http_status_code):
-    """Return and log error for Django Rest Framework API calls"""
-    logger.error(error_message)
+    """Return a standardized error for Django Rest Framework API calls"""
     return Response({'Error': error_message}, status=http_status_code)
```

Comment:
I've removed logging from here because the messages it dumps do not indicate the actual place in code where the error occurred. Also, it may not be necessary to log all API errors, and definitely not at the same log level.